packages/ubuntu_bootstrap/lib/installer/installer_wizard.dart

```diff
@@ -75,7 +75,9 @@ class _InstallWizard extends ConsumerWidget {
         ].contains(route)) {
           return true;
         } else {
-          return ref.read(installerModelProvider).hasRoute(route);
+          return context.mounted
+              ? ref.read(installerModelProvider).hasRoute(route)
+              : false;
```

Comment:
As mentioned yesterday, this doesn't solve the issue in `installer_test.dart` but merely moves it to a different place, see [here](https://github.com/canonical/ubuntu-desktop-provision/actions/runs/8202230324/job/22432561368?pr=490#step:7:375).
It first appeared after #451, which we need to partially [revert](https://github.com/canonical/ubuntu-desktop-provision/pull/484/commits/94c2c8b241ac9a76c16d61fc23e36f7ed88a55fe#diff-537df1c6577cdeacdbe4f4c502f6f51a573c63eb6ecb24f6df9571519b2a580f) to enable the UI reset for the autoinstall page.